### DISCUSSION TITLE
Group Assignment #2-2: Storyboard

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#33] [Assignments]
Instructions [here](https://github.com/yohman/21S-DH151/blob/main/Group%20Assignments/GroupAssignment2.md)

Include:

- Group name
- Group members
- Link to storyboard in your group repo

**Comment by alicelu170** [comment]
- Rethink Recycle
- Members: Alice, Emily, Hozumi
- https://github.com/hoz-map/Rethink-Recycling/blob/main/storyboard.md

**Reply by yohman** [reply]
I am impressed with the group's solid progress to date. The datasets are very well thought out, and each appear to be "legit" in a sense that they can be downloaded and integrated as map layers using javascript. The UX report also conveys the camaraderie of the team with respect to each other's ideas. The long scroll may need experimentation with CSS and other design frameworks like [bootstrap](https://getbootstrap.com/). For example, the [carousel template](https://getbootstrap.com/docs/5.0/examples/carousel/) may fit your design scheme. I also like the idea of having three separate maprooms that strategically inform your general narrative.

**Comment by ivanma9** [comment]
- Happy Tweet
- Members: Ivan, MJ, Josephine
- https://github.com/ivanma9/HappyTweet/blob/main/Group%20Assignments/Storyboard.md

**Reply by yohman** [reply]
While I like the ideas generated through the mindmap and crazy 8 exercise, I worry that the proposed datasets are not adequate to fulfill the group's technical scope and design ideas. The twitter dataset comprises of a million records of twitter locations as lat/lon's, with no additional metadata (so no user info, date, twitter message, etc). Mapping this data reveals a rather inconsistent pattern, with locations clustered around equi-distant square blocks around the world. Here is a preview of the mapped data (using [kepler.gl](https://kepler.gl/))

<img src="https://raw.githubusercontent.com/yohman/21S-DH151/main/images/worldtweets.png">

A zoomed in view:

<img src="https://raw.githubusercontent.com/yohman/21S-DH151/main/images/worldtweetszoom.png">

The second happiness dataset comprises of a separate table per year, by country. So it is essentially a single point for every country in the world with various indexes that measure "happiness." The correlation between these two datasets is very loose. I can foresee quantifying tweets by country, to get a tweet per country measure, but without knowing what these tweets are of, and how they were collected, it is difficult to come up with any relevant conclusion (unless I am missing something here!).

I propose that the group rethinks the data that will drive the project forward. Do you have alternative datasets you were looking at? Are there other inquiries of "happiness" that may be locationally aware that could be integrated into the project?

**Comment by kimjee8955** [comment]
- Datart
- Members: Annmarie Cong, Eustina Kim, Michelle Lee
- https://github.com/kimjee8955/DH151-Datart/blob/main/storyboard.md

**Reply by yohman** [reply]
Beautiful wireframes! The proposed datasets combined will map more than 1500 art pieces in Los Angeles. I wonder, however, how the group intends to push beyond the "points on the map," and make it experiential, or add some aspects of the flaneur (stroller) to the user experience (beyond a simple random art start). This will need to be explored with the group (it was not conveyed in the UX report) some more. What added layers of information, narratives, or interactive elements can you add to the project that will make it compelling beyond an archival map?

**Comment by justisg** [comment]
The Musical Maps
Members: Daniel, Kai, and Justis
https://github.com/justisg/TheMusicalMaps/blob/main/storyboard.md

**Reply by yohman** [reply]
I like how the group has strengths that are each respected and valued. The proposed storyboard has a nice map with a categorical and temporal filter, but I do not see how other aspects will be incorporated. Will the site provide additional knowledge beyond a categorial and temporal filter of music?

In lieu of this, do the three proposed datasets allow the group to do what is proposed? While I am unsure of whether or not this is the intended approach (it is not clear in the UX report or the project proposal), I wonder if the API calls are to be in "real time", meaning that a user can enter any artist, which would in turn activate the call to the three API's (songkick, spotify, and twitter) and produce artist specific maps, songs, and related tweets. How does this then inform the larger inquiry of the group, to see if modern changes in society have affected our taste in music?

**Comment by hanarama** [comment]
- Environmental Racism in Carceral California
- Members: Lisa Kahn, Hannah Lien, Julia Tanenbaum 
- Storyboard: https://github.com/jmtanenbaum/DHEnviroIncarceration/blob/main/storyboard.md

**Reply by lisalou36** [reply]
Data Update: https://github.com/jmtanenbaum/DHEnviroIncarceration/blob/main/Updated%20Data%20Plan.md

**Reply by yohman** [reply]
Your UX designs are beautiful and impressive. This is a very intriguing and possibly explosive/fascinating project. Has the group consulted with Professor Dolovich? If not, I encourage you to do so. The data provided by the [project GitHub repo](https://github.com/uclalawcovid19behindbars/data/blob/master/latest-data/adult_facility_covid_counts.csv) is fascinating, with a series of covid relevant indicators. I believe there is a lot the group can do with this alone. The CalEnviroScreen data can be used as a base layer of information, but I wonder about the direct correlation with the prison based data. What do you hope to inform the user by overlaying them on top of one another? Does the environment data directly inform on the conditions of the facilities in question?

What really piqued my attention is what you refer to as the "Featured Stories." Your report reveals very little about how you intend to produce these narratives, but it would certainly be an opportunity to humanize the project, and take it to another level of importance and urgency.

**Comment by derekwang99** [comment]
Ocean Pollution Posse
Members: Julia Wood, Eli Aranita, Derek Wang
https://github.com/eliaranita1/project1/blob/main/Group%20Assignments/%232/storyboard.md

**Reply by yohman** [reply]
While I would have liked to see a further breakdown of what each design element will do, the storyboard collection wireframes really does give the group's intentions justice. I am especially intrigued by the possibility of "The Life of Plastic," providing a spatial narrative on a plastic's journey from creation to the ocean. The opening touch is also enticing (everybody loves a little quiz!) and the activism component gives the project true meaning.

The proposed datasets all seem to be in workable format for the project. My question would be on how the group can integrate the data to provide a narrative that is not already provided by the provider's visualization pages (https://ourworldindata.org/plastic-pollution and https://globalfishingwatch.org/map/).

**Comment by zonaliao** [comment]
State-of-the-Climate
Members: Alekhya Vittalam, Natasha Grace, Zona Liao
https://github.com/alekhyavittalam/state-of-the-arts/blob/main/Week4/storyboard.md

**Reply by yohman** [reply]
The mindmaps + crazy 8's have really helped your group solidify a direction and purpose for the project. The resulting story boards are clear and clean. The proposed datasets are quite expansive, though carefully thought out and certainly within the realm of possibilities for the intended project scope. What would make your site unique, from what already exists on the provider website dashboards? How is your narrative different, and can that narrative on why pollution levels and income are correlated be clearly articulated through the user interaction? Thinking beyond the clickable points and colors would be the challenge moving forward.

**Comment by alexthegreat1** [comment]
Group: cARTographers
Members: Kai Colorado & Alex Haddad
Storyboard: https://github.com/kaicolorado/DH151-Group/blob/main/Group%20Assignments/Storyboard.md

**Reply by yohman** [reply]
I like how the sketches lead to the development of the wireframes that dictate the overall look and feel of your site. The UX report is missing the interactive elements and annotations to describe the different sections of the wireframes. I worry that the site, in its current form, lacks a narrative component, and relies on users toggling layers on/off. The proposed datasets are mostly at the state level. Thinking about your user, who are they, and what is the urgency and usefulness of the knowledge provided by your site? Thinking about it from the user's perspective may help the group address additional layers and/or interactivity to the data.

**Comment by frida-barragan** [comment]
Group: Health Mappers
Members:  Frida & Kate
Storyboard: https://github.com/frida-barragan/DH151_Group/blob/main/storyboard.md

**Reply by yohman** [reply]
I love how your mindmaps and crazy 8's are focused on the narrative, and why this is a matter of importance. It makes the user care about your project. There is a nice mix of historical/qualitative information and quantitative data through maps, and your wireframe is well annotated to bring this forth. The proposed datasets are all usable and mappable, combining polygons (service areas) with mental health facilities, and adding census data to inform policy. You can further append data to the service areas with the data from the DPH. I am also intrigued by Page 3 of your storyboard, which could bring a more qualitative emotional quality to your project.

**Comment by ainsleighdouglas** [comment]
Group: Team Data
Members: Ryan, Ainsleigh and Sebastien
Storyboard: https://github.com/ryanorlando/DH151-Project/blob/main/Assignment2/storyboard.md

**Reply by yohman** [reply]
Nice storyboarding, and I love the creativity in measuring our digital footprints through the "weight of data" widget. The proposed datasets are certainly usable within the scope of your project. I am still unsure on how the group intends to correlate these datasets to inform a narrative that makes sense. How do the locations of data centers affect global CO2 emissions? Is there literature that has exposed this as a known issue? I would also begin to think about the user experience when they navigate the site. Who is your user, and what will they gain from visiting your site? I believe adding more annotations and interactive documentation in the storyboards can help the team move forward with your vision.

**Comment by lisa58** [comment]
Group Name: Fantastic Four
Group Members: Labiba Alam, Haiqi Zhou, Adriana Romero, and Lynn Tieu
Storyboard: https://github.com/LynnT2/FantasticFour/blob/main/storyboard.md

**Reply by yohman** [reply]
The storyboards conveys the deep and thoughtful process the group has undergone. It is clean, narrative based, and hits the right balance of qualitative/quantitative data visualization. The timeline idea is great, prefacing the current situation with its historical legacy of inequity. Lynn's crazy eights serves double duty as a storyboard on its own.

Design-wise, the historical story cards look great as hovering boards on top of the map (rather than popup windows), and the arrows to navigate forward or backward provide a smooth user experience. Learning how to do this with function calls, time sliders, and floating divs will be a doable challenge for the group.

The proposed datasets will serve to fulfill the historical narrative. The NYTimes data will provide the current situation of Asian Hate Crimes. How the group combines these two datasets seamlessly will also be a challenge.
